src/routes/(shop)/bikes/+page.server.ts

```diff
@@ -1,7 +1,17 @@
 import { bikes as bikesService } from '$lib/server/db';
+import { redirect } from '@sveltejs/kit';
+import { hasEmptyUrlParams, clearEmptyUrlParams } from '$lib/utils';
+
+export async function load({ url }) {
+	if (hasEmptyUrlParams(url)) {
+		redirect(307, clearEmptyUrlParams(url).toString());
+	}
+
+	const showAll = url.searchParams.has('showAll') ?? '';
+	const filter = showAll ? '' : 'outOfShopDate = ""';
```

Comment:
I'd rather we define our own filter object that is unaware of which DB integration is being used, and therefore has no pocketbase specific functionality, so that we can keep the Services abstract, and each DB implementation can just expose the same interface.

But this can be handled in a follow up pr.